### ISSUE TITLE
td_player_id is null in some cases after updating to latest dev_version

### ISSUE THREAD

**Issue by iorparask** [nflverse/nflfastR#371]
For example:

pbp <- build_nflfastR_pbp('2022_01_JAX_WAS')
pbp %>% filter(!is.na(td_player_name)) %>% select(td_player_id)

\# A tibble: 6 × 1
  td_player_id
  <chr>       
1 00-0033282  
2 NA          
3 00-0035831  
4 00-0035831  
5 00-0035659  
6 NA    

there are several other cases where this happened for the games after 2022-09-11 after updating from dev version 9004 to 9006.

**Comment by mrcaseb** [comment]
The problem are missing player ids of rookies in the underlying data provided by the NFL. 
We hope they will add them soon.

**Comment by iorparask** [comment]
Thank you for the reply, however I would like to mention that I noticed this behaviour after updating to 9006. I had kept data downloaded with 9004 in a database and I saw that the field that had value became null.
For exaample,
play_id = 4232
game_id = 2022_01_JAX_WAS

In the data downloaded with 9004 I have:
td_player_name	td_player_id
J.Dotson	00-0037741

and in those downloaded with 9006 I have:
td_player_name	td_player_id
J.Dotson	NULL

So it appears that td_player_id was there before. Could it be an NFL issue?
